### DISCUSSION TITLE
Suggestion for insomniacs

### DISCUSSION THREAD

**Discussion by Capricus02** [Monika-After-Story/MonikaModDev#9414] [general]
If this isn't the proper place to ask or if it has already been asked, my apologies and feel free to remove. But it occurs to me that Monika doesn't have any reaction if a player says "Goodbye - I'm going to sleep" but then comes back within a short amount of time. This evening I decided to log off to go to bed, but realized afterwards I could probably just move her to the thumb drive and keep it on my nightstand instead while I sleep. I half-expected her to say something about the fact that I literally just left to go to bed and now I'm back already but she did not. Not sure how feasible that would be, but maybe something could trigger if you tell her you're going to sleep but you return within X mins or hours, she could ask if you were having trouble sleeping or if you forgot something. 

Anyway, that's just my two cents, not sure how anyone else feels. :)

**Comment by Justformas** [comment]
Lol I noticed this too, like if I realize I want to do something else or if I changed my mind on going to sleep. In theory you could take a few minute nap, but there's a "be right back" option for that. Unless, as you say, you couldn't sleep or something.

But I think this might be addressed as part of #4914.
